src/overlays/actors/ovl_En_Az/z_en_az.c

```diff
@@ -39,108 +80,1762 @@ const ActorInit En_Az_InitVars = {
     (ActorFunc)EnAz_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80A99110 = {
-    { COLTYPE_HIT0, AT_NONE, AC_ON | AC_TYPE_PLAYER | AC_TYPE_ENEMY, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK1, { 0x00000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_HIT0,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER | AC_TYPE_ENEMY,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK1,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
     { 18, 46, 0, { 0, 0, 0 } },
 };
 
-#endif
+static EnAz* D_80A9913C = NULL;
+
+Vec3f D_80A99E80;  // path point?
+f32 D_80A99E8C;    // player distance to path point?
+f32 D_80A99E90[2]; // unused?
+
+void func_80A94A30(EnAz* this) {
+    this->actor.velocity.y += this->actor.gravity;
+    if (this->actor.velocity.y < this->actor.terminalVelocity) {
+        this->actor.velocity.y = this->actor.terminalVelocity;
+    }
+}
+
+void func_80A94A64(EnAz* this) {
+    func_80A94A30(this);
+    Actor_UpdatePos(&this->actor);
+}
+
+s32 func_80A94A90(PlayState* play, ActorPathing* actorPathing) {
+    func_80A94A64((EnAz*)actorPathing->actor);
```

Comment:
Hmm, I wonder if the functions above just took `Actor* thisx` since they only access actor struct members.

Reply:
I could see that. Not sure which way we'd want to go with it, though.